doc/source/indexing.rst

```diff
@@ -941,9 +943,9 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 Indexing with isin
 ------------------
 
-Consider the ``isin`` method of Series, which returns a boolean vector that is
-true wherever the Series elements exist in the passed list. This allows you to
-select rows where one or more columns have values you want:
+Consider the :meth:`~Series.isin` method of Series, which returns a boolean 
+vector that is true wherever the Series elements exist in the passed list. 
```

Comment:
generally like to use double-backticks around Series, DataFrame, Index where possible.